pawno/include/modules/src/weaponshot.inc

```diff
@@ -597,6 +600,19 @@ CheckVisualDamageTextDraws()
 				Iter_SafeRemove(PlayerHasTVisualDmg, cur, i); // Safely remove him from the iterator array.
 		}
 	}
+	
+    // Update Damage Bubble
+	foreach(new i : Player)
```

Comment:
Ok, I actually never used anything with iteration other than for all players. Will try to implement that.